content/en/blog/2022/knative.md

```diff
@@ -1,53 +1,112 @@
 ---
 title: Distributed tracing in Knative
+linkTitle: Tracing in Knative
```

Comment:
@pavolloffay - notice the shortened title, which appears in the blog left-nav. Hopefully you're ok with this.